src/js/utils/dom.js

```diff
@@ -49,10 +49,10 @@ export function htmlToParentElement(html) {
 }
 
 export function sanitizeScriptNodes(element) {
-    const scripts = element.querySelectorAll('script');
-    for (let i = 0; i < scripts.length; i++) {
-        const script = scripts[i];
-        script.parentNode.removeChild(script);
+    const nodes = element.querySelectorAll('script,object,iframe');
```

Comment:
any tests to update?

Reply:
I will add tests to [dom-tests.js](https://github.com/jwplayer/jwplayer/blob/master/test/unit/dom-test.js#L84)

Reply:
done